drivers/saul/saul_str.c

```diff
@@ -47,8 +47,10 @@ const char *saul_class_to_str(const uint8_t class_id)
         case SAUL_SENSE_COLOR:  return "SENSE_COLOR";
         case SAUL_SENSE_PRESS:  return "SENSE_PRESS";
         case SAUL_SENSE_ANALOG: return "SENSE_ANALOG";
+        case SAUL_SENSE_UV:     return "SENSE_UV";
         case SAUL_SENSE_OBJTEMP:return "SENSE_OBJTEMP";
         case SAUL_SENSE_COUNT:  return "SENSE_PULSE_COUNT";
+        case SAUL_SENSE_DISTANCE: return "SENSE_DISTANCE";
```

Comment:
I think this indentation should be fixed.

Reply:
That would require to adjust all lines, but ok

Reply:
Done